### PR TITLE
Update sweetalert to sweetalert2

### PR DESCRIPTION
**Pull request by oliverhermanni**
sweetalert ist no longer maintained. sweetalert2 is in active development and has much cleaner code and has some nice additional features like input boxes. No other changes are required, it's backward compatible.

More info here: https://limonte.github.io/sweetalert2/

### CODE REVIEW

File: src/InteractsWithSparkConfiguration.php [hunk 1]
```diff
@@ -49,9 +49,19 @@ protected function configExists()
      * Get the Spark configuration file path.
      *
      * @return string
+     * @throws Exception
      */
     protected function configPath()
     {
-        return $_SERVER['HOME'].'/.spark/config.json';
```
Comment (dpde) [inline src/InteractsWithSparkConfiguration.php:55]: Are these changes really necessary for updating sweetalert?

if not and if you fixing a bug not related to the sweetalert update, this should be in a different pull request.

**Comment by taylorotwell** [comment]
Please only send the changes you want to make. Don't make unrelated changes.

**Comment by oliverhermanni** [comment]
Oops, sorry, there was a hiccup.
